tracking/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+project(tracking)
```

Comment:
I think it's good practice to define any resulting product as project. In future we should add project directive in all other directories.